frontend/src/metabase/ui/components/inputs/Calendar/Calendar.styled.tsx

```diff
@@ -23,7 +23,7 @@ export const getCalendarOverrides = (): MantineThemeOverride["components"] => ({
         borderRadius: theme.radius.xs,
 
         "&:hover": {
-          backgroundColor: theme.fn.themeColor("bg-light"),
+          backgroundColor: "var(--mb-color-bg-light)",
```

Comment:
pretty much the same color, but I only change the style that I tests, so I leave the rest the same.